readme.md

```diff
@@ -112,6 +112,41 @@ The function throws an error if anything goes wrong, which can be handled by cat
 await mdToPdf({ content: '# Hello, World' }, { dest: 'path/to/output.pdf' });
 ```
 
+#### Table of content
+
+Place the following comment `<!-- TOC -->` where you want the table of content be generated from document headings
+
+Options for table of content are : 
+
+* `TOC_headings`: Set the table of content title ( default: 'Table of content' )
+* `TOC_depth`: Skip n level for the table of content generation ( default: 0 ) 
```

Comment:
```suggestion
* `toc_depth`: Skip n level for the table of content generation (default: 3)
```

I think `3` should be the default value, and the depth should determine how many levels to include (see next comment).

Reply:
Maybe it should be name toc_skip in this case. That was the original functionality. I need it to be able to provide a main title for the document without include it to the TOC. Maybe toc_depth can be add in another PR?

Reply:
Yeah the depth is how many levels deep the TOC will include the headings (usually 3 in scientific contexts), which is a common option for tools that can generate a TOC (e. g. LaTeX).

> I need it to be able to provide a main title for the document without include it to the TOC.

Yeah that makes sense... I was thinking maybe it makes sense that `h2` is treated as level 1 🤷🏻‍♂️

Reply:
I propose to change toc_depth by toc_skip for nom and set the default value to 1 so it skip the document title. we can add toc_depth later

Reply:
Ok. With flat structure i was able to easily  implement toc_skip and toc_depth. I will update the PR soon